BFWControls/Modules/Table/Controller/StaticTableViewController.swift

```diff
@@ -114,9 +99,21 @@ class StaticTableViewController: UITableViewController {
     // MARK: - UITableViewDelegate
 
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        let height = intrinsicHeightCells
+        var height = intrinsicHeightCells
             ? UITableViewAutomaticDimension
             : super.tableView(tableView, heightForRowAt: superIndexPath(for: indexPath))
+        // Check filledUsingLastCell attribute.
+        if filledUsingLastCell {
+            // Is this last cell?
```

Comment:
I don't think we need this comment and the one above. The lines they describe are fairly obvious.